manage.py

```diff
@@ -26,6 +26,21 @@ def make_shell_context():
 manager.add_command('db', MigrateCommand)
 
 
+def create_default_admin(first_name, last_name, email, password):
+    if (User.query.filter_by(email=email).first() is not None):
```

Comment:
Related: more pythonic would be `if not User.query.filter_by(email=email).first():` Whether it's more/less readable, I'll let you decide